App.js

```diff
@@ -1,18 +1,173 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Alert, AppRegistry, Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import Picker from 'react-native-picker';
+import { Dropdown } from 'react-native-material-dropdown';
+
+// export default class LogIn extends React.Component {
+//   // static navigationOptions = {
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
```

Comment:
is the text actually being used for anything? this should have all of the fields that you are updating- e.g. username, password, and typing on each input should have a corresponding function that updates the state. For an example, you can check the functions/state on the corresponding component on the web side